src/server/services/wms/qgswmsparameters.cpp

```diff
@@ -323,6 +323,18 @@ namespace QgsWms
     const QgsWmsParameter pHighlightBufferSize( QgsWmsParameter::HIGHLIGHT_LABELBUFFERSIZE );
     save( pHighlightBufferSize );
 
+    const QgsWmsParameter pLabelRotation( QgsWmsParameter::HIGHLIGHT_LABELROTATION, QVariant::Double );
+    save( pLabelRotation );
+
+    const QgsWmsParameter pLabelDistance( QgsWmsParameter::HIGHLIGHT_LABELDISTANCE, QVariant::Double );
+    save( pLabelDistance );
+
+    const QgsWmsParameter pLabelHali( QgsWmsParameter::HIGHLIGHT_LABELHALI );
```

Comment:
What do you think about using longer more readable identifiers? Something like `HIGHLIGHT_LABELHORIZONTALALIGNMENT`.

I would really prefer the even more readable form `HIGHLIGHT_LABEL_HORIZONTAL_ALIGNMENT` but I see that there are pre-existing compact forms `HIGHLIGHT_LABELBUFFERSIZE` without the`_`, maybe you could deprecate the pre-existing form and go for the better readability on all parameters.

Reply:
Yes, I was looking at the already existing parameter names. No strong opinion from my side if it should be HIGHLIGHT_LABELHORIZONTALALIGNMENT or HIGHLIGHT_LABEL_HORIZONTAL_ALIGNMENT.

Reply:
The latter is more readable, but I'm leaving the choice to you. 

If you think they are too long, a decent compromise could be something like `HIGHLIGHT_LABEL_HORIZ_ALIGN`.

Reply:
I'll go with HIGHLIGHT_LABEL_HORIZONTAL_ALIGNMENT